osh/client/commands/shortcuts.py

```diff
@@ -145,3 +145,12 @@ def upload_file(hub, srpm, target_dir, parser):
         return hub.upload_file(os.path.expanduser(srpm), target_dir)
     except Fault as e:
         handle_perm_denied(e, parser)
+
+
+def verify_scan_profile_exists(hub, profile_name):
+    """Verify if scan profile exists"""
+    profiles = hub.scan.list_profiles()
+    profile_exists = any(p["name"] == profile_name for p in profiles)
+    if profile_exists:
```

Comment:
There is no need to introduce a local variable that will be set and read only once.
```suggestion
    if any(p["name"] == profile_name for p in profiles):
```

Reply:
I originally also want a one liner. but some may say one liner ruins readability.

Anyway, I'm ok with getting rid of this accidentally introduced local vars. Thanks for the suggestion.